cromshell

```diff
@@ -1471,6 +1488,99 @@ function fetch-all()
   return 0
 }
 
+# Get the cost for a workflow ID:
+function cost()
```

Comment:
You get some strange errors if you run with a workflow ID that isn't in your files:

```
./cromshell cost workflow-I-don't-have
typeDATEat[1:210]
```

```
./cromshell cost-detailed workflow-I-don't-have
Error in query string: Error processing job 'broad-dsde-dev:bqjob_r40b2725b4fe1eddd_0000017ae9b77342_1': Could not cast literal "--" to type DATE at [10:32]
```

As discussed it should maybe get the dates from the metadata instead of the database.

Reply:
oops!  Yup - fixed.

And I added in the metadata checks.